color.go

```diff
@@ -80,3 +150,68 @@ func xParseColor(s string) color.Color {
 	}
 	return color.Black
 }
+
+func getMaxMin(a, b, c float64) (max, min float64) {
+	if a > b {
+		max = a
+		min = b
+	} else {
+		max = b
+		min = a
+	}
+	if c > max {
+		max = c
+	} else if c < min {
+		min = c
+	}
+	return max, min
```

Comment:
```suggestion
	return max(a, b, c), min(a, b, c)
```

Reply:
`min` and `max` require go1.21

Reply:
ah, right, could use `ordered.Min` and `ordered.Max` then... 

https://github.com/charmbracelet/x/blob/main/exp/ordered/ordered.go

I think it would look clearer

Reply:
I don't want to introduce more dependencies. I will put a todo note for now until we bump the go version

Reply:
fair